src/pages/home.js

```diff
@@ -637,7 +637,7 @@ const query = gql`
         }
       }
     }
-    recent: allCollectives(type: COLLECTIVE, orderBy: createdAt, orderDirection: DESC, limit: 4) {
+    recent: activeCollectives(orderDirection: DESC, limit: 4) {
```

Comment:
might need to change if you end up changing consolidating this query with `allCollectives(isActive: true, minBackerCount: 10)`